cmd/mattermost-mattermod/main.go

```diff
@@ -31,6 +39,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if migrateVersion != -1 {
+		err = runMigrations(config.DriverName, config.DataSource, migrateVersion)
```

Comment:
Nit: we could have separate main for this under something like `/cmd/mattermod-migrator`.

Reply:
Fair enough. I'll make the changes. :+1: